sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -875,19 +1443,21 @@ pi_result piEventGetInfo(pi_event, pi_event_info, size_t, void *, size_t *) {
 pi_result piEventGetProfilingInfo(pi_event Event, pi_profiling_info ParamName,
                                   size_t ParamValueSize, void *ParamValue,
                                   size_t *ParamValueSizeRet) {
-  (void)Event;
-  (void)ParamName;
-  (void)ParamValueSize;
-  (void)ParamValue;
-  (void)ParamValueSizeRet;
-  DIE_NO_IMPLEMENTATION;
+  std::cerr << "Warning : Profiling Not supported under PI_ESIMD_CPU"
+            << std::endl;
```

Comment:
Please, do not print debug info unconditionally. Suggest wrapping it to something similar to `zePrint` from `pi_level_zero.cpp` or converting to `assert`.

Reply:
Will replace with print style similar to `zePrint`.

Reply:
Done - `PI_SYCL_TRACE` value is referenced for conditional debug info printout.